mizar/dp/mizar/operators/droplets/droplets_operator.py

```diff
@@ -92,14 +93,56 @@ def assign_bouncer_droplet(self, bouncer):
         droplets = set(self.store.get_all_droplets())
         if len(droplets) == 0:
             return False
-        d = random.sample(droplets, 1)[0]
+        # Read cluster_gateway_host_ip from configmap
```

Comment:
Can we review this code in a zoom meeting tomorrow with Phu? I think that will be a lot more efficient.

Reply:
@jshaofuturewei @vinaykul please add me to the code review meeting if needed.